### PR TITLE
Fix Javadoc in PathPattern

### PR DESCRIPTION
**Pull request by gurpiarbassi**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@gurpiarbassi Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=24841)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-framework&pullRequestId=24841) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@gurpiarbassi Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=24841)!

File: spring-web/src/main/java/org/springframework/web/util/pattern/PathPattern.java [hunk 1]
```diff
@@ -59,10 +59,10 @@
  * <li><code>/resources/{&#42;path}</code> &mdash; matches all files
  * underneath the {@code /resources/} path and captures their relative path in
  * a variable named "path"; {@code /resources/image.png} will match with
- * "spring" &rarr; "/image.png", and {@code /resources/css/spring.css} will match
- * with "spring" &rarr; "/css/spring.css"</li>
+ * "path" &rarr; "/image.png", and {@code /resources/css/spring.css} will match
+ * with "path" &rarr; "/css/spring.css"</li>
  * <li><code>/resources/{filename:\\w+}.dat</code> will match {@code /resources/spring.dat}
- * and assign the value {@code "spring"} to the {@code filename} variable</li>
+ * and assign the value {@code "filename"} to the {@code filename} variable</li>
```
Comment (sbrannen) [inline spring-web/src/main/java/org/springframework/web/util/pattern/PathPattern.java:65]: This was actually correct.

```suggestion
 * and assign the value {@code "spring"} to the {@code filename} variable</li>
```

**Review by sbrannen — CHANGES_REQUESTED** [review]

**Comment by sbrannen** [comment]
This has been merged into `master`.

Thanks

**Pull request merged**